default/README.md

```diff
@@ -0,0 +1,48 @@
+# Resource directory (default)
+
+The contents of the resource directory define the assets available to
+FreeOrion, as well as logic for modifiable content.
+These assets determine the appearance and functionality of FreeOrion.
+
+While much of the content within this directory is required in any resource
+directory, most is either
+* required only from inclusion in other entries
+* strictly optional
+
+Each sub-directory has an accompanying README file within it.
+This file should give a guideline for any entries or files required by
+FreeOrion if a new resource directory is created.
+
+An alternate resource directory can be specified with the
+--resource-dir command line flag, or by changing the resource-dir node in the
+user's config.xml.
+See http://www.freeorion.org/index.php/Config for details on the config.xml
+file.
+
+## Contents
+
+* customizations/  -  User customizations
+* data/  -  Contains data files used by FreeOrion, including graphics, fonts,
+and sound.
+* python/  -  All content processed by python scripts are located here.  This
+is largely logic for AI decisions, events that occur each turn, and creation
+of a new universe when starting a new game.
+* scripting/  -  This directory contains the definitions of content within
```

Comment:
Remove the word "largely".  The list is exhaustive.  

Although in the future the list might not be exhaustive, the sentence is stronger without "largely".

Style.